Modules/LearningSequence/classes/content/class.ilObjLearningSequenceContentGUI.php

```diff
@@ -112,6 +105,22 @@ protected function confirmDelete()
 		$this->tpl->setContent($confirmation->getHTML());
 	}
 
+	protected function delete()
+	{
+		$ref_ids = $_POST["id"];
+
+		$ref_ids = array_map(function($i) {
+			return (int)$i;
+		}, $ref_ids);
+
+		$rep_utils = new ilRepUtil();
```

Comment:
Ab hier bitte ins Objekt selbst, das ist Funktionalität, die nicht in die GUI soll.